internal/inspect/table_index_sizes/table_index_sizes.go

```diff
@@ -18,7 +18,7 @@ SELECT c.relname AS table,
   pg_size_pretty(pg_indexes_size(c.oid)) AS index_size
 FROM pg_class c
 LEFT JOIN pg_namespace n ON (n.oid = c.relnamespace)
-WHERE n.nspname NOT IN ('pg_catalog', 'information_schema')
+WHERE n.nspname NOT IN ('pg_catalog', 'information_schema', 'auth', 'realtime', 'pgsodium', 'supabase_migrations', 'vault', 'storage')
```

Comment:
Why are we moving some queries but not this one?

Can we also split the refactor into a separate commit so it's easier to review what's changed?

There's a full list of supabase managed schemas defined in misc.go. I think we should use it instead.

Reply:
missed the removal of that query 🙏